code/game/machinery/vending/vendor_types/requisitions.dm

```diff
@@ -337,11 +348,14 @@
 			return //We found our item, no reason to go on.
 
 /obj/structure/machinery/cm_vending/sorted/cargo_ammo/cargo/blend
-		icon_state = "req_ammo_wall"
-		tiles_with = list(
-			/obj/structure/window/framed/almayer,
-			/obj/structure/machinery/door/airlock,
-			/turf/closed/wall/almayer)
+	icon_state = "req_ammo_wall"
+	vend_delay = 5
```

Comment:
You will want to try to reduce this delay (meaning you'll need a faster animation) since req generally vends things quickly. Alternatively you somehow queue it up, but I don't recommend this.

Reply:
You can vend up to 3 items per second with this delay, so i think it's fine

Reply:
Played with it, delay 3 works too without me needing to adjust anything else